regparser/citations.py

```diff
@@ -10,7 +10,7 @@
 
 class Label(object):
     #   @TODO: subparts
-    _p_markers = tuple('p{0}'.format(i) for i in range(1, 10))
+    _p_markers = tuple('p{0}'.format(i) for i in range(1, 11))
```

Comment:
I can't see how this is being used. But, however it's being used, it should be changed to allow these to not be hard coded. If it's meant to match a `p<number>` a regex might work. If it's mean to generate them it should be a function, if it's meant to generate them in order it should be an iterator.